drivers/adc/adc_ads1x1x.c

```diff
@@ -138,11 +141,39 @@ struct ads1x1x_data {
 	int16_t *buffer;
 	int16_t *repeat_buffer;
 	struct k_thread thread;
+	k_tid_t	tid;
 	bool differential;
+	struct gpio_callback gpio_cb;
+	struct k_work work;
 
 	K_THREAD_STACK_MEMBER(stack, CONFIG_ADC_ADS1X1X_ACQUISITION_THREAD_STACK_SIZE);
 };
 
+#ifdef CONFIG_ADC_ADS1X1X_TRIGGER
+static inline void setup_rdy_pin(const struct device *dev, bool enable)
```

Comment:
nit: naming of function doesn't seem to be consistent with the others, maybe change name to `ads1x1x_setup_rdy_pin`

Reply:
done